File: ffxicomp-client/src/models/GearItem.ts

export interface GearStat {
  name: string;
  value: number;
}

export interface GearItem {
  id: number;
  name: string;
  category?: string;
  stats: GearStat[];
  jobs: string[];   // e.g. ["WAR", "SAM", "NIN"]
  slots: string[];  // e.g. ["Main", "Sub", "Ammo"]
}

File: Api/Models/Dto/GearItem.cs
namespace FFXIComp.Api.Models.Dto;

public class GearItemDto
{
    public int Id { get; set; }
    public string Name { get; set; } = null!;
    public string? Category { get; set; }
    public List<GearStatDto> Stats { get; set; } = [];
    public List<string> Jobs { get; set; } = [];         // e.g., ["WAR", "NIN"]
    public List<string> Slots { get; set; } = [];        // e.g., ["Main", "Sub"]
}

